code/api/src/modules/userProducts/types.js

```diff
@@ -4,17 +4,17 @@ import UserType from '../user/types'
 import ProductType from '../product/types'
 
```

Comment:
Seems touchy about camelcasing and thank you for your diligence on pluralization.